lib/browser-vm/context.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const contextOptionsSymbol = Symbol('contextOptionsSymbol');
+
+/**
+ * options - is supported
+ */
```

Comment:
We need no this comments, please describe it not PR instead if there are some important difference in behavior.